src/Controller/Admin/MemberCrud.php

```diff
@@ -107,6 +114,55 @@ public function cancelMembership(AdminContext      $adminContext,
         return $this->redirect($redirectUrl);
     }
 
+    public function convertMemberToSupportMember(AdminContext $adminContext, AdminUrlGenerator $adminUrlGenerator, EntityManagerInterface $entityManager): Response
+    {
+        /**
+         * @var Member
+         */
+        $member = $adminContext->getEntity()->getInstance();
+
+        // check if current user is allowed and the selected member is allowed to be removed and converted
+        $redirectUrl = $adminUrlGenerator
+            ->setController(self::class)
+            ->setAction(Crud::PAGE_EDIT)
+            ->setEntityId($member->getId())
+            ->generateUrl();
+
+        $supportMember = new SupportMember();
+        $supportMember->setFirstName($member->getFirstName());
+        $supportMember->setLastName($member->getLastName());
+        $supportMember->setEmail($member->getEmail());
+        $supportMember->setPhone($member->getPhone());
+        $supportMember->setIban($member->getIban());
+        $supportMember->setAddress($member->getAddress());
+        $supportMember->setCity($member->getCity());
+        $supportMember->setPostCode($member->getPostCode());
+        $supportMember->setCountry($member->getCountry());
+        $supportMember->setDateOfBirth($member->getDateOfBirth());
+        $supportMember->setRegistrationTime(new DateTime()); // now seems more logically
```

Comment:
Ik zou hier de oorspronkelijke registratie datum behouden. Want, uit de geboortedatum kunnen wij herleiden wanneer iemand van lid naar steunlid is gegaan (het jaar dat ze 28 zijn geworden), dus die hoeven we niet per se op te slaan. De oorspronkelijke registratie datum daarentegen kunnen we niet meer terug krijgen als we het lid verwijderen.